src/main/resources/spring/camel-context.xml

```diff
@@ -59,6 +59,26 @@
                     <bean ref="initialSavingsEstimationReportService" method="findReportSummaryById(${header.id})" />
                 </route>
             </get>
+            <delete uri="/{id}">
+                <description>Delete a report</description>
+                <route id="report-delete">
+                    <bean ref="analysisService" method="findById(${header.id})"/>
+                    <choice>
+                        <when>
+                            <simple>${body} != null</simple>
+                            <bean ref="analysisService" method="deleteById(${header.id})" />
```

Comment:
As agreed we could change the approach to return no content and `204` HTTP code (`No Content`) in case the delete request worked successfully.
In this way we are also compliant with [RFC-2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.7)

Reply:
@mrizzi my last commit should make the endpoint return 204 HTTP code with an empty body. Thanks for noticing this